src/components/ChatApp/TopBar.react.js

```diff
@@ -90,38 +86,6 @@ class TopBar extends Component {
       search: false,
     });
 
-    let url;
-
-    if (cookies.get('loggedIn')) {
-      url = `${
-        urls.API_URL
-      }/aaa/showAdminService.json?access_token=${cookies.get('loggedIn')}`;
-      $.ajax({
-        url: url,
-        dataType: 'jsonp',
-        jsonpCallback: 'pfns',
-        jsonp: 'callback',
-        crossDomain: true,
-        success: function(newResponse) {
-          let showAdmin = newResponse.showAdmin;
-          cookies.set('showAdmin', showAdmin, {
-            path: '/',
-            domain: cookieDomain,
-          });
-          this.setState({
-            showAdmin: showAdmin,
-          });
-          // console.log(newResponse.showAdmin)
-        }.bind(this),
-        error: function(newErrorThrown) {
-          console.log(newErrorThrown);
-        },
-      });
-
-      this.setState({
-        showAdmin: cookies.get('showAdmin'),
-      });
-    }
     // Check Logged in
     if (cookies.get('loggedIn')) {
```

Comment:
I did not edit this part in my PR because I knew you were working on this block of code. Again, use the state value instead of cookies.

Reply:
@AakashMallik hey, I guess this could be taken look in a new PR. This branch has issue while resolving conflicts. So would be better to take it up in a new PR, anyways this is for getAdmin.

Reply:
@fragm3 I think I can easily make this change in my PR https://github.com/fossasia/chat.susi.ai/pull/1813, as I have taken care of state hydration there and passing the props to TopBar. So if this PR gets merged first, i'll make this change in my PR.

Reply:
Okay great!